rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf

```diff
@@ -702,7 +701,6 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
 	setvar:tx.%{rule.id}-OWASP_CRS/WEB_ATTACK/XSS-%{matched_var_name}=%{tx.0}"
 
 #
-# http://openmya.hacker.jp/hasegawa/security/utf7cs.html
```

Comment:
That's a shame this is down (http://web.archive.org/web/20170126151511/http://openmya.hacker.jp/hasegawa/security/utf7cs.html) maybe we should mirror it?